test/CMakeLists.txt

```diff
@@ -385,27 +385,29 @@ if (TARGET TBB::tbb)
     if (WIN32)
         tbb_add_test(SUBDIR tbb NAME test_numa_dist DEPENDENCIES TBB::tbb)
     endif()
+    if (NOT TBB_EMSCRIPTEN)
+        tbb_add_test(SUBDIR tbb NAME test_partitioner DEPENDENCIES TBB::tbb)
+	tbb_add_test(SUBDIR tbb NAME test_parallel_invoke DEPENDENCIES TBB::tbb)
+	tbb_add_test(SUBDIR tbb NAME test_parallel_pipeline DEPENDENCIES TBB::tbb)
+	tbb_add_test(SUBDIR tbb NAME test_task_group DEPENDENCIES TBB::tbb)
+	tbb_add_test(SUBDIR tbb NAME test_task_arena DEPENDENCIES TBB::tbb)
```

Comment:
There are a lot of disabled tests for common oneTBB functionality. Can you explain in the description of this PR, what exactly doesn't work in these tests and why we should completely disable them instead of fixing/workardound/disable particular not working test case?

P.S. Could you please update PR name to be a little bit more descriptive?

Reply:
@JhaShweta1 The first question here is still relevant.

Reply:
I have added the reasons in the description. Except malloc tests, we had identified reason behind all the test failures. A few of of the failed test passes with the additional flags, that I added in the TBB_flags. Other tests that either pass conditionally/malloc test are disabled - description for those is in listed. We were only interested in the list of APIs that work for emscripten and therefore we have not  investigated failure of malloc tests